File: sdk/python/entity/recon_order_split_sub.py
from dataclasses import dataclass
from datetime import datetime


@dataclass
class ReconOrderSplitSub:
    """对账订单分账子记录"""
    id: int = None
    order_no: str = None
    sub_order_no: str = None
    merchant_id: str = None
    split_amount: float = None
    status: int = None
    create_time: datetime = None
    update_time: datetime = None

File: sdk/go/entity/recon_order_split_sub.go
package entity

import (
	"time"

	"github.com/shopspring/decimal"
)

// ReconOrderSplitSub 对账订单分账子记录
type ReconOrderSplitSub struct {
	ID           int64           `json:"id" gorm:"primaryKey"`
	OrderNo      string          `json:"orderNo"`
	SubOrderNo   string          `json:"subOrderNo"`
	MerchantId   string          `json:"merchantId"`
	SplitAmount  decimal.Decimal `json:"splitAmount" gorm:"type:decimal(20,2)"`
	Status       int             `json:"status"`
	NotifyStatus int             `json:"notifyStatus"`
	NotifyResult string          `json:"notifyResult"`
	CreateTime   time.Time       `json:"createTime"`
	UpdateTime   time.Time       `json:"updateTime"`
}
